content/news/013.dd-ix-opening.de.md

```diff
@@ -0,0 +1,23 @@
+---
+published: 2024-11-28
+description: Vorträge, Snacks und viel gute Stimmung
+title: Die Feierliche Eröffnung des DD-IX
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 013_dd_ix_opening_press.webp
+---
+
+Am 25. November feierten wir unsere offizielle Eröffnung mit Freunden des DD-IX's und lokalen Internetakteuren. Die Veranstaltung fand 
+im SachsenEnergieCenter in der Nähe des Dresdner Hauptbahnhofs statt. 
+
+[Rack](013_dd_ix_opening_rack.webp)
+
+Wir möchten uns bei unseren Vortragenden: Prof. Matthias Wählisch, Prof. Thomas Popp, Thomas Eibeck, Prof. Thomas Horn, Vint Cerf, Christian Dieckhoff,
```

Comment:
Reihenfolge finde ich unguenstig. Alphabetisch statt in der Reihenfolge der Vortraege (die Reihenfolge kennt der Leser nicht). 

Hinter "Vortragenden" kein Doppelpunkt.

Reply:
done, sorted by family name